pages/annotation.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+
+    <b-container>
+
+        <b-row>
+            <h1>Column Annotation</h1>
+        </b-row>
+
+        <b-row>
+            <b-col cols="4"></b-col>
+            <b-col cols="4">
+                <img class="vertical-center-mockup"
+                     :src="require(`~/assets/${mockupImage.filename}`)"
+                     :width="mockupImage.width"
+                     :height="mockupImage.height"/>
+            </b-col>
+            <b-col cols="4"></b-col>
+        </b-row>
+
+    </b-container>
+
+</template>
+
+<script>
+
+    export default {
+
+        data() {
+
+            return {
+
+                mockupImage: {
+
+                    filename: "annotation_mockup.jpg",
+                    height: 956,
+                    width: 1045
+                }
+            }
+        }
+    }
+
+</script>
+
+<style>
```

Comment:
Does it make sense for the css to live in its own corner of the project? I'm worried that if each page has a stylesheet section it may become tricky to keep them in sync. This is just for demo now, but "nothing is as permanent as a temporary solution" (someone, probabably)

Reply:
Global styles can be stored and made accessible using this addition to the nuxt.config.js file. See: https://nuxtjs.org/docs/configuration-glossary/configuration-css/#style-extensions

Typically I would tend to list global styles in a file like that and local ones per component if they are not going to be used elsewhere. 

In Nuxt, the latter is accomplished by including the 'scoped' keyword as a boolean attribute in the style tag. (https://vue-loader.vuejs.org/guide/scoped-css.html)

Reply:
@surchs We can go ahead and adopt that change if you want. We'll make a global css file in the assets/css folder.